server/src/modules/rules/getter/radarr-getter.service.spec.ts

```diff
@@ -167,6 +167,93 @@ describe('RadarrGetterService', () => {
     });
   });
 
+  describe('fileAudioLanguages', () => {
+    let collectionMedia: CollectionMedia;
+    let plexLibraryItem: PlexLibraryItem;
+
+    beforeEach(() => {
+      collectionMedia = createCollectionMedia(EPlexDataType.MOVIES);
+      collectionMedia.collection.radarrSettingsId = 1;
+      plexLibraryItem = createPlexLibraryItem('movie');
+      tmdbIdService.getTmdbIdFromPlexRatingKey.mockResolvedValue({
+        type: 'movie',
+        id: 1,
+      });
+    });
+
+    it('should return audio languages', async () => {
+      const movie = createRadarrMovie({
+        movieFile: createRadarrMovieFile({
+          mediaInfo: {
+            audioBitrate: 1000,
+            audioChannels: 2,
+            audioCodec: 'AAC',
+            audioLanguages: 'eng',
+            audioStreamCount: 1,
+            videoBitDepth: 8,
+            videoBitrate: 5000,
+            videoCodec: 'H.264',
+            videoFps: 24,
+            resolution: '1920x1080',
+            runTime: '02:00:00',
+            scanType: 'Progressive',
+            subtitles: 'eng',
```

Comment:
I think it would be cleaner to be able to merge in some mediaInfo defined here, as we're only interested in setting the **audioLanguages** property. The remaining properties can be left to faker.